File: www/cgi-bin/type_echo.py
#!/usr/bin/env python3
import os

print('Content-Type: text/plain; charset=utf-8')
print('X-Galaxy-CGI: python')
print('')
print('cgi_type=py')
print('method=' + os.environ.get('REQUEST_METHOD', ''))
print('query=' + os.environ.get('QUERY_STRING', ''))

File: www/cgi-bin/type_echo.php
<?php
header('Content-Type: text/plain; charset=utf-8');
header('X-Galaxy-CGI: php');
echo "cgi_type=php\n";
echo "method=" . ($_SERVER['REQUEST_METHOD'] ?? '') . "\n";
echo "query=" . ($_SERVER['QUERY_STRING'] ?? '') . "\n";
?>
